### PR TITLE
Implemented mandatory headers

### PR DESCRIPTION
**Pull request by osher-sade**

### CODE REVIEW

**Review by itaykl — APPROVED** [review]

File: test/integration/server/dal/connection-manager.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { ConnectionOptions, createPolarisConnection } from '@enigmatis/polaris-typeorm';
-import { polarisGraphQLLogger } from './logger';
+import { polarisGraphQLLogger } from '../utils/logger';
```
Comment (DoctorVoid) [inline test/integration/server/dal/connection-manager.ts:2]: Does a logger accounts as a util anyways?

Comment (osher-sade) [reply test/integration/server/dal/connection-manager.ts:2]: Not really, should we create a specific folder for it?

**Review by DoctorVoid — APPROVED** [review]

File: test/integration/server/dal/connection-manager.ts (hunk 1 above)
Comment (DoctorVoid) [reply test/integration/server/dal/connection-manager.ts:2]: I'm not sure... Well lets leave it like this in the meanwhile.

**Pull request merged**